components/chainhook-cli/src/service/mod.rs

```diff
@@ -567,7 +571,7 @@ pub struct StreamingData {
 pub struct ExpiredData {
     pub number_of_blocks_evaluated: u64,
     pub number_of_times_triggered: u64,
-    pub last_occurrence: u128,
+    pub last_occurrence: String,
```

Comment:
What is the rationale for migrating to a String?

Reply:
I have a comment explaining this weirdness here: https://github.com/hirosystems/chainhook/blob/44fedf2a87be76d419e2ec4d6c2e36f21cf93ea0/components/chainhook-cli/src/service/mod.rs#L550-L552